bot/controllers/addGames.js

```diff
@@ -0,0 +1,30 @@
+// module.exports = () => {
+// const util = require('apex-util');
+//
+// const _run = (message) => {
+//     const disallowedRoles = ['admin', 'armada officers', 'armada officer', 'moderator', 'privateers', 'privateer', 'tester', 'crew', 'fleet officer', '@everyone'];
```

Comment:
Consider removing this as disallowedRoles does not seem to apply to your logic implemented.